extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.Dialog.scss

```diff
@@ -43,7 +43,8 @@
 	max-height: 480px;
 }
 
-#ve-ui-mwSaveDialog-captcha > div {
+#ve-ui-mwSaveDialog-captcha > div,
+#ve-ui-mwSaveDialog-captcha > p {
```

Comment:
Is this still needed or can we leave it at div since you updated the fancy captcha template to have a {{div}} wrapper?